jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverInfo.java

```diff
@@ -44,7 +44,15 @@
     public JFunc1<Capabilities, WebDriver> getRemoteDriver;
 
     public boolean isLocal() {
-        return isEmpty(DRIVER.remoteUrl) && (isNotBlank(DRIVER.path) && isNotBlank(path.execute()) || downloadType != null);
```

Comment:
change to
return DRIVER.remoteRun;
and place verifications on DRIVER.remoteUrl, DRIVER.path, path, and downloadType into appropriate setup methods

Reply:
Please see my comment to previous CR above ^^^